custom/enikshay/tasks.py

```diff
@@ -523,7 +524,7 @@ def _get_fulfilled_vouchers(self):
         relevant_vouchers = [
             voucher for voucher in self._get_all_vouchers()
             if (voucher.get_case_property('voucher_type') == 'prescription'
-                and voucher.get_case_property('state') == 'fulfilled')
+                and voucher.get_case_property(DATE_FULFILLED))
```

Comment:
@proteusvacuum is this a reasonably property to check?  This could also check for a state within a list of all states "fulfilled" or later